module-2/backend/week_three.md

```diff
@@ -10,14 +10,56 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+
+rails new <project_name>
+
 2. What do Models generally inherit from in rails?
+
+ApplicationRecord
+
 3. What do Controllers generally inherit from in a rails project?
-4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+ApplicationController
+
+4. How would I create a route if I wanted to see a specific horse in my routes file assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+def show
+  @horse = Horse.find(params[:id]
+ end
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+
+rake routes - prefix, verb, URI patters and controller action
+
 6. What is an example of a route helper? When would you use them?
+
+routes helpers are all the prefixes in the rake routes display. you dont have to use them but it may be preferable to typing in the whole URI pattern manually.
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+
+i seem to be able to use them interchangeably
+
```

Comment:
_path helpers provide a site-root-relative path. You should probably use this most of the time. i.e. "/horses" Internal to my app
_url helpers provide an absolute path, including protocol and server name. i.e "localhost:3000/horses" Accessing from browser